yanic/yanic.conf.tmpl

```diff
@@ -12,9 +12,16 @@ synchronize      = "1m"
 # how often request per multicast
 collect_interval = "1m"
 
+# all
 [respondd.sites.ffmuc]
 domains             = [{% for site, domains in sites.items() %}{% for domain in domains %}"{{site}}_{{ domain }}", {% endfor %}{% endfor %}]
 
+# ffdon
+[respondd.sites.ffdon]
+domains             = ["ffdon_mitte", "ffdon_nordwest", "ffdon_sued"]
```

Comment:
We could try something like this to automatically populate the sites from netbox if we ever add another one to ffdon:
```suggestion
domains             = [{% for domain in sites["ffdon"] %}"ffdon_{{ domain }}", {% endfor %}]
```

Reply:
sounds good to me